app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="lists-items">
+   <%= image_tag(category.icon, alt: 'Example Image') %>
+   <p><%= category.name %></p>
+   <p>Total:$ <%= category.total_transaction_amount %></p>
+</div>
+
```

Comment:
- The error on the screenshot arises when I tried to create a new category. I think you should equally make sure your app works correctly. I tried compiling the app once and again but the issue repeated. Let's kindly fix it :+1: 
![Screenshot from 2023-12-01 16-30-30](https://github.com/Kccendana/budget_App/assets/63087491/8953e3fe-7c39-4f52-b2a8-c2da5274be9d)